empress/reconcile/recon_vis/utils.py

```diff
@@ -265,11 +284,14 @@ def uniquify(elements):
     """
     return list(set(elements))
 
-def build_temporal_graph(host_tree, parasite_tree, reconciliation):
+def build_temporal_graph(host_tree, parasite_tree, reconciliation, add_strong_constraints = True):
```

Comment:
I think the name `use_strong_constraints` might be more indicative. This is very minor though. Feel free to keep as is.

Reply:
Should the `constraints` be plural or singular `constraint`?

Reply:
when we use strong constraints, we basically add two more constraints for every transfer event, so I think add_strong_constraints is more indicative.

Constraints are plural to be consistent with the comments in the code